it/uk/gov/hmrc/partnershipidentificationfrontend/api/controllers/JourneyControllerISpec.scala

```diff
@@ -280,7 +280,7 @@ class JourneyControllerISpec extends ComponentSpecHelper with JourneyStub with A
 
         post("/partnership-identification/api/" + apiUrlSuffix, testJourneyConfigJson ++ testIncomingJsonWelshLabels)
 
-        val expectedWelshLabels = Some(JourneyLabels("This is a welsh service name"))
+        val expectedWelshLabels = Some(JourneyLabels(Some("This is a welsh service name"), None))
```

Comment:
We're not currently testing when the english labels block is sent to us. Can we add a test, or update a current test to have the english as well? Could potentially add it to these tests "POST a correct json with labels to /api/<url suffix PartnershipType specific>"